packages/core/src/screens/Settings.tsx

```diff
@@ -218,6 +220,18 @@ const Settings: React.FC<SettingsProps> = ({ navigation }) => {
         },
       ]
     }
+    const additionalSection = settingsSections.find((item) => item.header.title === t('Settings.AppSettings'))
+    if (additionalSection && !additionalSection.data.some((item) => item.testID === testIdWithKey('Logout'))) {
+      additionalSection.data = [
+        ...additionalSection.data,
+        {
+          title: t('Settings.Logout'),
+          accessibilityLabel: t('Settings.Logout'),
+          testID: testIdWithKey('Logout'),
+          onPress: () => lockOutUser(reasonLogout),
+        },
+      ]
+    }
```

Comment:
```suggestion
        },
        {
          title: t('Settings.Logout'),
          accessibilityLabel: t('Settings.Logout'),
          testID: testIdWithKey('Logout'),
          onPress: () => lockOutUser(reasonLogout),
        },
      ]
    }
```

Would you be ok with this?